content/applications/productivity/whatsapp.rst

```diff
@@ -0,0 +1,22 @@
+========
+WhatsApp
+========
+
+**WhatsApp** is a popular messaging app that enables users to send texts, make calls, and share
+media securely and conveniently. Businesses can use `WhatsApp Business
+<https://business.whatsapp.com>`_ to communicate with their customers, send documents and provide
```

Comment:
The link for this is https://developers.facebook.com/products/whatsapp/ 
The above link is for normal WhatsApp business

Reply:
Thank you @jva-odoo  :)